pkg/authenticator/config/config.go

```diff
@@ -143,5 +148,17 @@ func populateConfig() (*Config, error) {
 		defaultConfig.ClientCertPath = envVal
 	}
 
+	// Parse client cert timeout if one is provided from env
+	defaultConfig.ClientCertTimeout = DefaultClientCertTimeout
+	clientCertTimeoutString := os.Getenv("CONJUR_CLIENT_CERT_TIMEOUT")
+	if len(clientCertTimeoutString) > 0 {
+		parsedClientCertTimeout, err := time.ParseDuration(clientCertTimeoutString)
+		if err != nil {
+			return nil, log.RecordedError(log.CAKC034E, err.Error())
+		}
+
+		defaultConfig.ClientCertTimeout = parsedClientCertTimeout
+	}
```

Comment:
we'd want to add some unit tests to this before it lands on master, if we end up merging this